### DISCUSSION TITLE
1kb size limit is too low for json metafields

### DISCUSSION THREAD

**Discussion by nboliver** [Shopify/function-examples#222] [Feedback: General Improvements]
After banging my head against the wall trying to figure out why an input metafield (`paymentCustomization`) was returning null even though I could see it using an admin API query, I found this limitations doc: https://shopify.dev/docs/apps/functions/input-output#limitations

Use case is validating the customer's shipping address against a large set of valid locations that are stored in JSON. Increasing the limit to 10kb would be beneficial - 1kb doesn't store many entries even with a simple JSON schema!

**Comment by nickwesselman** [comment]
Thanks @nboliver! We have some updates planned for this limit.

**Comment by nickwesselman** [comment]
This limit is now 10Kb!

https://shopify.dev/changelog/shopify-functions-input-limit-updates
